app/gui/view/examples/component-list-panel-view/src/lib.rs

```diff
@@ -198,8 +200,23 @@ pub fn main() {
         let panel = app.new_view::<ide_view_component_list_panel::View>();
         scene.layers.node_searcher.add(&panel);
         panel.show();
+
+        let breadcrumbs = app.new_view::<Breadcrumbs>();
+        breadcrumbs.set_y(400.0);
+        breadcrumbs.frp().set_size(Vector2(500.0, 100.0));
+        breadcrumbs.set_entries_from((
+            vec![
+                Breadcrumb::from("home"),
+                Breadcrumb::from("data"),
+                Breadcrumb::new_with_icon("read", icon::Id::DataInput),
+            ],
+            0,
+        ));
+        breadcrumbs.set_base_layer(&app.display.default_scene.layers.node_searcher);
+
+
```

Comment:
Please note that in #7350 breadcrumbs were moved to the documentation view (and to the documentation demo-scene), so there would be a merge conflict in this place.